plugins/flytekit-vscode/flytekitplugins/vscode/decorator.py

```diff
@@ -92,45 +146,68 @@ def download_vscode(
 
     # Download remote file to local
     code_server_tar_path = download_file(code_server_remote_path, DOWNLOAD_DIR)
+    
+    plugin_paths = []
+    for plugin in plugins_remote_paths:
```

Comment:
We should use the plugin in Open-VSX instead of the official vscode marketplace due to legal reason. Refer to the [codeserver doc](https://coder.com/docs/code-server/latest/FAQ#why-cant-code-server-use-microsofts-extension-marketplace):
> Because of this, we can't offer any extensions on Microsoft's marketplace. Instead, we use the [Open-VSX extension gallery](https://open-vsx.org/), which is also used by various other forks. It isn't perfect, but its getting better by the day with more and more extensions.

You can find plugins on [OpenVSX](https://open-vsx.org/) . For example:
python plugin: https://open-vsx.org/api/ms-python/python/2023.20.0/file/ms-python.python-2023.20.0.vsix
jupyter plugin: https://open-vsx.org/api/ms-toolsai/jupyter/2023.9.100/file/ms-toolsai.jupyter-2023.9.100.vsix
Note: you can find the exact link by "right click" and "copy link address" in the plugin homepage.

Reply:
Thanks for suggestion, updated pr